Containerfile

```diff
@@ -1,6 +1,5 @@
 # vim: set filetype=dockerfile
-ARG LIGHTSPEED_RAG_CONTENT_IMAGE=quay.io/openshift-lightspeed/lightspeed-rag-content@sha256:8adbb6387204d39fb58eda8d7cdaab6407bb8e09d3050854588f89cd0af62077
-
+ARG LIGHTSPEED_RAG_CONTENT_IMAGE=quay.io/openshift-lightspeed/lightspeed-rag-content@sha256:cbc9b0ea7c96fe400f2f8a8470976dd2c953c8e6239dc61b6aba2fad267363cd
```

Comment:
This is a `ARG`.  Please update it in the workflows, e.g. the push workflow is: https://github.com/ansible/ansible-chatbot-service/blob/main/.tekton/ansible-chatbot-service-push.yaml#L269

This ContainerFile will be a generic one on the upstream, we use the `ARG` to control what we need to use

Reply:
yeah but only if we wanted to override the value with a custom one. in this case I am suspecting of a meta issue and by default this should be running without a problem. 

What if someone wants to build the image without the pipeline? for the dev environmetn this should be sth already valid as well so I will insist on keeping the change. If this doesnt work, then I can revert it back to tthe upstream one as I will look at the problem from a different angle.

Reply:
The default in the container file should be empty. And a local build can supply the value. Build ARG is part of the spec supported by both podman and docker.

That said I'll lgtm this for you

Reply:
I agree. however my aim is keeping the changes in the container file as min as it needed. once we have these pipelines running we can focus on the other things I guess by improving the Containerfile and stuff around those.